backend/globaleaks/rest/requests.py

```diff
@@ -15,8 +15,8 @@
 user_roles_regexp                 = r'^(admin|custodian|receiver)$'
 user_states_regexp                = r'^(enabled|disabled)$'
 receiver_img_regexp               = r'^([a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}).png$'
-email_regexp                      = r'^([\w-]+\.)*[\w-]+@([\w-]+\.)+[a-z]{2,9}$|^$'
-email_regexp_or_empty             = r'^([\w-]+\.)*[\w-]+@([\w-]+\.)+[a-z]{2,9}$|^$'
+email_regexp                      = r'^(([\w-]\.){0,100}[\w-]{1,100}@([\w-]+\.){0,100}[\w-]{1,100})$|^$'
```

Comment:
The `email_regexp` variable needs the concluding `|^$` for emails to be accepted in UI. This may be because the variable `email_regexp_or_empty` should be used in the python somewhere where it is not currently.